test/techHttpSpec.js

```diff
@@ -37,7 +57,7 @@ describe("tech-http", function() {
 
         return http.get("avop-ruuid-42", "http://foo:8080/api/bar/baz").then(function(result) {
             assert.ok(result.foo === "bar", "request apis should have been wrapped");
-            assert.ok(parseFloat(duration.ms) > 100, "the request time should have been logged");
+            assert.ok(parseFloat(duration.ms) > 90, "the request time should have been logged");
```

Comment:
Because the mockRequet uses a setTimeout to simulate a server that responds in 100ms. I had a weird bug once where the test failed because of the duration, so 10ms less will do.